docker/bin/vernemq.sh

```diff
@@ -101,10 +101,11 @@ fi
 
 if env | grep -q "VERNEMQ_ENABLE_SSL_LISTENER"; then
     # Let's do our magic. First of all, let's ask for certificates.
-    if ! curl -s -d '{"label": "primary"}' -X POST $CFSSL_URL/api/v1/cfssl/info | jq -e -r ".result.certificate" > /etc/ssl/cfssl-ca-cert.crt; then
+    if ! res=`curl -s -d '{"label": "primary"}' -X POST $CFSSL_URL/api/v1/cfssl/info`; then
```

Comment:
This would protect from the API not responding, but not from the response being different from what we expect (e.g. not containing `result.certificate`).

`jq` actually fixed [the problem](https://github.com/stedolan/jq/pull/1697) in a later version, but for now I would manage this with the (stronger) check that our certificate isn't empty, so:
```bash
    cacert=$(curl -s -d '{"label": "primary"}' -X POST $CFSSL_URL/api/v1/cfssl/info | jq -e -r ".result.certificate")
    if [ -z "$cacert" ]; then
        echo "Could not..."
        ...
    fi
    echo $cacert > /etc/ssl/cfssl-ca-cert.crt
```